company/teammate_qualities.md

```diff
@@ -0,0 +1,52 @@
+# Teammate qualities
```

Comment:
I wonder if this list is intended to be actionable? If yes, for whom?

I'm curios about the qualities because some of them mention skills (e.g. empathy) but some are more of a statement "You believe in our mission and are motivated".

I think I would personally derive more value from a list of "Skills we consider essential" because it will hint which skills I should develop next.

+ I wonder if we should mention "Diversity" in general and making space for "Diversity of opinions" in particular.
To be honest the list below seems very prescriptive of qualities :)

Reply:
I think I understand (and share) your concerns, @twop, but at the same time, I also think that a "prescriptive" list of traits isn't necessarily a bad thing. It can serve as a guide, as something we all aspire towards. We just need to make sure that it's inclusive enough and nobody feels excluded simply by reading this and thinking "oh, so I'll never be this ideal teammate".

Reply:
To play devil's advocate and try to illustrate a possible different perception  (not necessarily mine):
I read the list and cannot help but think: "it is not me". Therefore, "me" right now is not good enough, so I must be someone else, which can apply immense pressure.

Also a lot of points can be flipped:
> Proactive: You do not wait for others to tell you what to do

To be proactive two things must be true:
- You believe that your ideas are good
- You believe that your actions will matter, e.g. somebody cares about what you think and do.

Both of this points are context dependent. Aka it is hard to be proactive in the environment when you are told what to do (aka you need to execute what **I** think is right)

So In my head, "Proactive" is a function of the culture not a "trait" that you can develop with no context. In other words, if you trust people to do their best work "Proactivity score" will go up.

> Responsible: and humbly admit when you could have done better

That is definitely disirable but might be challenging if there is an expectation to do everything "the right way", e.g. no room for mistakes and experiments. In my head, "Responsibility score" can be increased by a) making space for mistakes b) let people own their learnings, which again, context dependent.

I hope that illustration makes sense.

Hot take: my personal preference would be to either remove that part completely or rephrase it in this way:  "traits we strive to develop", and then bullet points should describe why we think they are important for the company.

Reply:
Yes, agree. I think an additional paragraph that puts this into context would help.

I guess I did have some implicit context already, because I didn't read the list and thought "Oh, I don't do X, that's a problem." Instead, when I for example read this:

 > You deliver the right quality at the right time; you don’t over-engineer or under-engineer a solution.

I thought "haha, yeah, buddy, I wish" because _that_ is IMHO one of the hardest things in software engineering. But at the same time I also thought "that is a fantastic goal. I really want to be able to do that better and I'm actively working on getting better at it".

That just as an explanation on why I don't think "prescriptive" is per se a bad thing :)